lambda/github_api.py

```diff
@@ -0,0 +1,97 @@
+import os
+import time
+
+import requests
+
+import config
+from logger import LOG
+
+TOKEN_PREFIX = "/alert-processor/tokens/github/"
+API_ROOT = "https://api.github.com"
+ACCESS_TOKEN = None
+
+
+def set_github_access_token(token):
+    """ Get the access token once """
+    global ACCESS_TOKEN
+    ACCESS_TOKEN = token
+
+
+def get_github_access_token():
+    """ Get the access token once """
+    global ACCESS_TOKEN
+    if ACCESS_TOKEN is None:
```

Comment:
`if not ACCESS_TOKEN`?

Reply:
Access token is actually set to `None` on line 12. This is the most specific check. I always prefer positive matches as opposed to negating conditions.